source/opentelemetry-collector-contrib/exporter/azuremonitorexporter/channels.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019, OpenTelemetry Authors
```

Comment:
I think, we should figure out pulling azure monitor exporter as package rather than code and that we way we can update easily without much work and help us maintain easily.